packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -152,6 +152,8 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Fri Mar 30 2018 Justin Haynes <jushay@amazon.com> - 1.17.3-1
+- Cache Agent vesion 1.17.3
```

Comment:
I think https://github.com/aws/amazon-ecs-init/pull/160 warrants a changelog as well (as an `Enhancement`).

Reply:
Yep! I think we may want to adopt the `CHANGELOG.md` procedure that the agent follows and port that to package specific changelogs at release time.

Reply:
Agreed. For now, I'll add `- Use s3client instead of httpclient when downloading` to the spec file.

Reply:
Added an issue to track this thought: https://github.com/aws/amazon-ecs-init/issues/162